YogaKit/Source/YGLayout.h

```diff
@@ -10,9 +10,25 @@
 #import <UIKit/UIKit.h>
 #import <yoga/YGEnums.h>
 
+@class YGLayout;
+
+@protocol YGLayoutEntity <NSObject>
+@property (nonatomic, readonly, strong) YGLayout *yoga;
```

Comment:
I'd tend to agree with @fjolnir : it's more in line with CocoaTouch to have them as properties (see `UIViewAnimating` for example). The only reason I have used for defining properties as methods is to indicate that they are not `O(1)`.